README.md

```diff
@@ -71,17 +71,23 @@ In order to quickly set up all components, execute the following commands:
 NAMESPACE=kubeflow
 kubectl create namespace ${NAMESPACE}
 
+# Which version of Kubeflow to use
+# For a list of releases refer to:
+# https://github.com/kubeflow/kubeflow/releases
+VERSION=v0.1.0-rc.0
```

Comment:
Do we want to set rc.0 if this is last thing to do before actual 0.1.0?:) Also we should document how to use master besides pinned version.

Reply:
I wanted to put a tag that exists. We can update it in a subsequent change once we have an actual release.